Updates for side-by-side and web root references

wadepickett opened the pull request:
Updated links for side-by-side and web root references
Fixes https://github.com/aspnet/AspNetCore.Docs/issues/10559

tdykstra commented on aspnetcore/fundamentals/index.md:
```diff
@@ -264,6 +264,8 @@ For more information, see [Content root](xref:fundamentals/host/web-host#content
 
 The web root (also known as *webroot*) is the base path to public, static resources, such as CSS, JavaScript, and image files. The static files middleware will only serve files from the web root directory (and sub-directories) by default. The web root path defaults to *{Content Root}/wwwroot*, but a different location can be specified when [building the host](#host).
 
+For more information, see [Web root](xref:fundamentals/host/web-host#webroot).
```
This close to 3.0 GA, I would link to generic-host rather than soon-to-be-deprecated web-host.  You'll have to use a regular relative link that specifies 3.0.

wadepickett replied:
Thanks Tom!  I will add a moniker range to point to ContentRootPath on the .NET Generic Host doc for 3.0 and point to the Web root for earlier versions.

tdykstra commented on aspnetcore/includes/benefits.md:
```diff
@@ -11,5 +11,5 @@ ASP.NET Core provides the following benefits:
 * A lightweight, [high-performance](https://github.com/aspnet/benchmarks), and modular HTTP request pipeline.
 * Ability to host on [IIS](xref:host-and-deploy/iis/index), [Nginx](xref:host-and-deploy/linux-nginx), [Apache](xref:host-and-deploy/linux-apache), [Docker](xref:host-and-deploy/docker/index), or self-host in your own process.
-* Side-by-side app versioning when targeting [.NET Core](/dotnet/articles/standard/choosing-core-framework-server).
+* Side-by-side app versioning when targeting [.NET Core](/dotnet/standard/choosing-core-framework-server#a-need-for-side-by-side-of-net-versions-per-application-level).
```
The link text ".NET Core" is misleading now for the target.  I would put this link on the phrase "side-by-side app versioning" and you could remove the link from ".NET Core"

wadepickett replied:
Submitted a correction for this, thanks Tom!

tdykstra commented on aspnetcore/index.md:
```diff
@@ -57,7 +57,7 @@ There are several advantages to targeting .NET Core, and these advantages increa
 * Cross-platform. Runs on macOS, Linux, and Windows.
 * Improved performance
-* Side-by-side versioning
+* [Side-by-side app versioning](/dotnet/standard/choosing-core-framework-server#a-need-for-side-by-side-of-net-versions-per-application-level)
```
I would omit "app" from the link text.  It's about .NET versions that are running sxs, not app versions.

wadepickett replied:
Submitted a correction for this, thanks Tom!

wadepickett commented:
@tdykstra
Added changes per your review and updated description of what is being handled in this issue verses what was moved out to new issues for updating additional topics we are linking to.  Thanks for the help!

wadepickett commented:
Do not merge.  There was an issue with the webroot bookmark.  Fixing that now.

tdykstra commented on aspnetcore/fundamentals/index.md:
```diff
@@ -264,6 +264,18 @@ For more information, see [Content root](xref:fundamentals/host/web-host#content
 
+::: moniker range=">= aspnetcore-3.0"
+
+For more information, see [ContentRootPath](/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.0#contentrootpath)
```
content root is a different thing than web root -- the 3.0 link should go to web root as well, just in the generic-host doc.

wadepickett replied:
Just saw your message show up as I merged and the screen redrew!  fixing!
